UAVPose[] q
float64[] steadyTime
nav_msgs/OccupancyGrid map
